CONTRIB.rst

```diff
@@ -116,6 +116,7 @@ please open an issue on GitHub or mention it on the mailing list.
 - Erik Weßels <https://github.com/BioNij>
 - Erik Whiting <https://github.com/erik-whiting>
 - Fabian Egli <https://github.com/fabianegli>
+- Fabio Zanini <https://github.com/iosonofabio>
```

Comment:
You can add yourself to the NEWS file too, perhaps also add a line about the new functionality too?

Reply:
yep done